packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx

```diff
@@ -19,10 +19,16 @@ export class InstantAppsControlPanel {
 
   @Element()
   el: HostElement;
-
+  
+  /**
+   * Determine which widgets or components to display in the control panel
+   */
   @Prop()
   components: ControlPanelComponent[] = [];
-
+ 
+  /**
+   * MapView or SceneView
```

Comment:
Suggested edit: A reference to the MapView or SceneView.

Reply:
This similar to how the ArcGIS Maps SDK for JS docs the `view` property

Reply:
Sounds good, I will update that and include it when I update more of the md files. Thanks!